code/parse/sexp.cpp

```diff
@@ -39013,6 +39065,14 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 		"\t1:\tBoolean, whether or not to enable sensor static\r\n"
 	},
 
+	{OP_HUD_FORCE_EMP_EFFECT, "hud-force-emp-effect\r\n"
+		"\tActivates or deactivates emp effect for the player."
+		"Takes 2 or more Arguments...\r\n"
+		"\t1:\tNumber, emp intensity (0 to 500)\r\n"
+		"\t2:\tNumber, emp duration in seconds\r\n"
```

Comment:
Millisecond durations would be useful.  It can just be divided by 1000 before being passed to `emp_start_local`.

Reply:
what would be the point? `emp_start_local` takes seconds so it'd be rounded anyway.

Reply:
It takes seconds, but the seconds are specified as a `float`.  So passing it 1.5 should be valid.